webpack/assets/javascripts/react_app/redux/actions/hosts/hosts.test.js

```diff
@@ -0,0 +1,22 @@
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import * as actions from './index';
+import immutable from 'seamless-immutable';
+import { requestData, onFailureActions } from './hosts.fixtures';
+const mockStore = configureMockStore([thunk]);
+
+describe('hosts actions', () => {
+  it(
+    'creates HOST_POWER_STATUS_REQUEST and fails when nock is not applied',
```

Comment:
HTTP mocking lib, since this action triggers a AJAX request, nock can intercept and create a response. if it is not used, error will be returned, and this is the case i'm testing.